src/pages/Index/Section/Personal/index.js

```diff
@@ -1,32 +1,59 @@
 import React, { useState, useEffect } from 'react'
+import { useHistory } from 'react-router-dom'
 import StartGuide from './StartGuide'
 import Cabinet from './Cabinet'
 import { Section, Container, Row } from '../index'
 import { Tab } from '../Trends'
 import Toggle from '../../../../components/VisibilityIndicator/Toggle'
 import styles from './index.module.scss'
 
+const tabByAnchor = () => {
+  const hash = window.location.hash
+
+  if (hash === TabType.CABINET.anchor) {
+    return TabType.CABINET.title
+  }
+}
+
```

Comment:
1. Function declaration is stylistically better for multi-line function body;
2.  `TabType` is used before declaration, which might be a little bit confusing;
3. Function names should start with a verb.

In this case it's better to have a backlinking object
```js 
const HashTab = Object.entries(TabHash).reduce((acc, [key, value]) => {
  acc[value] = key
  return acc
}, {})
```